app/views/admin/overview/_health_check.html.erb

```diff
@@ -0,0 +1,24 @@
+<section class="crayons-card p-7 m:basis-1-3">
+  <h2 class="crayons-subtitle-1 mb-4">Health check</h2>
+
+  <a href="<%= admin_privileged_reactions_path %>" class="flex flex-col c-link c-link--block -mx-4">
+    <strong class="fs-xl"><%= @flags_count %></strong>
+    <span>Flags raised by <span class="fw-bold">Moderators</span></span>
+    <small class="opacity-75">
+      <%= pluralize(@flags_posts_count, "post") %>,
+      <%= pluralize(@flags_comments_count, "comment") %>,
+      <%= pluralize(@flags_users_count, "user") %>
+    </small>
+  </a>
+
+  <a href="<%= admin_feedback_messages_path %>" class="flex flex-col c-link c-link--block -mx-4">
+    <strong class="fs-xl"><%= @open_abuse_reports_count %></strong>
+    <span><%= "Report".pluralize(@open_abuse_reports_count) %> submitted by <span class="fw-bold">Users</span></span>
```

Comment:
Here we call the `String#pluralize` method while further up we use the `pluralize` helper. I think we should stick to a single style.

```suggestion
    <span><%= pluralize(@open_abuse_reports_count, "Report") %> submitted by <span class="fw-bold">Users</span></span>
```

Reply:
I actually reverted this one: `pluralize` adds actual value in front of the label by default and we didn't want that hence the different style..

Reply:
Ah, sorry, I didn't realize you were using them in different ways.